llvm/lib/Target/RISCV/RISCVInstrInfoXCheri.td

```diff
@@ -1562,6 +1564,13 @@ defm : CheriStPat<store, CFSD, FPR64>;
 
 } // Predicates = [HasCheri, HasStdExtD, IsCapMode]
 
+let Predicates = [HasCheri, HasStdExtF, IsRV64, NotCapMode] in {
+// TODO: double patterns for RV32 (which requires BuildPairF64 and SplitF64)
+// TODO: We may want to add int<->float patterns for pointers in capability mode.
```

Comment:
This belongs with F not D (it applies to both, but F is really the core place for it).

Reply:
I presume you mean the comment not the patterns?